rpc/provider/proxy.ts

```diff
@@ -28,7 +34,13 @@ export const proxyProvider: Provider<string, Event, Event, Event> = (url, listen
       })()
     },
     release: () => {
-      const { cleanUp, listeners, inner } = connection(url, listener)
+      let conn
+      try {
+        conn = connection(url, listener)
```

Comment:
if the connection was not created in `.send`, this call will attempt to create it again.
And if it fails to create the connection it will throw.